core-tests/src/test/java/org/mule/runtime/core/registry/RequestContextTestCase.java

```diff
@@ -6,13 +6,29 @@
  */
 package org.mule.runtime.core.registry;
 
+import static java.time.OffsetTime.now;
+import static java.util.Optional.empty;
 import static org.junit.Assert.assertEquals;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 import static org.mule.runtime.core.DefaultMuleEvent.setCurrentEvent;
 import static org.mule.tck.MuleTestUtils.createErrorMock;
 
 import org.mule.runtime.core.DefaultMessageContext;
+import java.nio.charset.Charset;
```

Comment:
Why all these additional imports when there are no other changes to class? Was this not compiling?